README.md

```diff
@@ -305,6 +306,16 @@ Example can be found under `./example/crs/mellanox.com_v1alpha1_macvlannetwork_c
 
 A deployment example can be found under `example` folder [here](https://github.com/Mellanox/network-operator/blob/master/example/README.md).
 
+## Docker image
+Network operator uses `alpine` base image by default. To build Network operator with
+another base image you need to pass `BASE_IMAGE` argument:
+
+```
+docker build -t network-operator \
+--build-arg BASE_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:latest \
```

Comment:
you need to add to Travis to build both type of images (not for this release)

Reply:
We'll move it to jenkins to not hit dokerpull limit. golang image from dokcerhub is still using for base builder image